ui-admin/src/study/families/FamilyRouter.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import {
+  Link,
+  Route,
+  Routes
+} from 'react-router-dom'
+import { StudyEnvContextT } from '../StudyEnvironmentRouter'
+import { NavBreadcrumb } from 'navbar/AdminNavbar'
+import { StudyEnvParams } from '@juniper/ui-core'
+import FamilyView from './FamilyView'
+import {
+  userHasPermission,
+  useUser
+} from 'user/UserProvider'
+
+/** routes to family views as appropriate */
+export default function FamilyRouter({ studyEnvContext }: {studyEnvContext: StudyEnvContextT}) {
+  const { user } = useUser()
+
+  if (!userHasPermission(user, studyEnvContext.portal.id, 'prototype')) {
+    return <p>You do not have the appropriate permissions to access this prototype feature.</p>
+  }
+
+  return <>
+    <NavBreadcrumb value={studyEnvContext.currentEnvPath}>
+      <Link to={`${studyEnvContext.currentEnvPath}/participants`} className="me-1">
+          participants</Link>
```

Comment:
Give some indication on the breadcrumbs that were in a family path
```suggestion
          participants</Link> family
```

Reply:
I made it `families` and in the PR where it I will incorporate families into the participant list, I'll make sure that when you use the link it will pre-group by family